less/gmcq.less

```diff
@@ -179,6 +179,11 @@
         }
     }
 
+    .gmcq-item-graphic-attribution {
+        font-size:0.75em;
+        line-height: 1em;
```

Comment:
@lc-thomasberger & @kirsty-hames, are we alright to introduce `em`s as a unit in core plugins? Can only recall seeing pixel values previously.